src/integrationTest/resources/application-integration-test.yml

```diff
@@ -6,6 +6,15 @@ management.endpoint:
   info.cache.time-to-live: 0
 
 spring:
+  jpa:
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.H2Dialect
+    show-sql: true
+    generate-ddl: true
+    hibernate:
+      ddl-auto: create
```

Comment:
This was required to get the IT in question (`should fail to create induction given induction already exists()`) to work. I can't say I fully understand why yet, or why more tests weren't failing.